ERCS/erc-7616.md

```diff
@@ -0,0 +1,263 @@
+
+### Metadata
+
+#### Metadata Extensions
+
+ERC-7616 metadata extensions are compatible ERC-721 metadata extensions. You can see more details in [ERC-721](./eip-721.md).
+
+## Rationale
+
+### Design decision: Value transfer from token to address
+
+The 'value' of a token is a property of the token and is not linked to an address, so to transfer the value to an address would be actually transferring it to a token owned by that address, not the address itself.
+
+From the implementation perspective, the process of transferring values from token to address could be done as follows: (1) create a new token for the recipient's address, (2) transfer the value to the new token from the 'source token'. So that this method is not fully independent from the ID-to-ID transfer method, and can be viewed as syntactic sugar that wraps the process described above.
+
+Developer can also choose to do (1) find one token owned by the address (2) transfer the value to the found token.
+
+Both implementations described above should be treated as compliant with this standard.
+
+The purpose of maintaining id-to-address transfer function is to maximize the compatibility with most wallet apps, since for most of the token standards, the destination of token transfer are addresses. This syntactic wrapping will help wallet apps easily implement the value transfer function from a token to any address.
+
+### Design decision: Relationship between different approval models
+
+For semantic compatibility with ERC-721 as well as the flexibility of value manipulation of tokens, we decided to define the relationships between some of the levels of approval like that:
+
+1. Approval of an id will lead to the ability to partially transfer values from this id by the approved operator; this will simplify the value approval for an id. However, the approval of total values in a token should not lead to the ability to transfer the token entity by the approved operator.
+2. `setApprovalForAll` will lead to the ability to partially transfer values from any token, as well as the ability to approve partial transfer of values from any token to a third party; this will simplify the value transfer and approval of all tokens owned by an address.
+
+## Backwards Compatibility
+
+As mentioned in the beginning, this EIP is backward compatible with ERC-20.
+
+## Reference Implementation
+
+- [ERC-3525 implementation](../assets/eip-3525/contracts/ERC3525.sol)
+
+## Security Considerations
+
+The value level approval is isolated from ERC-721 approval models, so that approving value should not affect ERC-721 level approvals. Implementations of this EIP must obey this principle.
+
+Since this EIP is ERC-721 compatible, any wallets and smart contracts that can hold and manipulate standard ERC-721 tokens will have no risks of asset loss for ERC-7616 tokens due to incompatible standards implementations.
```

Comment:
```suggestion
Since this EIP is ERC-721 compatible, any wallets and smart contracts that can hold and manipulate standard ERC-721 tokens will have no risks of asset loss for hybrid fungible tokens due to incompatible standards implementations.
```